app/(home)/@posts/loading.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
+import PostSkeleton from '@/src/compositions/post/post-skeleton';
 
-export default async function Loading() {
-    return <>Loading Page Posts</>;
+export default function LoadingPost() {
+    return <PostSkeleton classNames="h-[400px] w-full" />;
```

Comment:
maybe add more than one post skeleton here? like 3 for example, since it's the home page where multiple posts will be shown as soon as loaded